GameServer/commands/gmcommands/announce.cs

```diff
@@ -92,7 +122,8 @@ public void OnCommand(GameClient client, string[] args)
 					{
 						foreach (GameClient clients in WorldMgr.GetAllPlayingClients())
                             if(clients != null)
-							    clients.Out.SendDialogBox(eDialogCode.SimpleWarning, 0, 0, 0, 0, eDialogType.Ok, true, LanguageMgr.GetTranslation(clients, "GMCommands.Announce.ConfirmAnnounce", client.Player.Name, message));
+	                            // Message: Announce from {0}: {1}
+	                            ChatUtil.SendTypeMessage((int)eMsg.DialogWarn, client, "GMCommands.Announce.ConfirmAnnounce", message);

```

Comment:
[02:17:04,208] [ERROR] [DOL.Language.LanguageMgr] [Language-Manager] Parameter number incorrect: GMCommands.Announce.ConfirmAnnounce for language EN, Arg count = 1, sentence = 'Announce from {0}: {1}', args[0] = 'test'

Reply:
After [c09fa47](https://github.com/OpenDAoC/OpenDAoC-Core/pull/30/commits/c09fa4721843aecadac25e9c3f153a651958b5ba):
`InvalidCastException` line 97.

Reply:
Sorry, still troubleshooting things. My test server runs on a separate iMac from the MacBook I work on, so the changes committed aren't quite complete yet.